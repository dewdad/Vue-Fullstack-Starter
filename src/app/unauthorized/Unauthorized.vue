<template>
  <v-container fluid fill-height>
    <v-layout justify-center align-center>
      <div class="text-xs-center">
        <div class="display-2 primary--text pa-3">Whoops, 401!</div>
        <div class="pa-2">You don't have permission to go to this page.</div>
        <v-btn to="/" color="primary" outline>Get me out of here!</v-btn>
      </div>
    </v-layout>
  </v-container>
</template>

<script>
// @flow

export default {
  metaInfo: {
    title: 'Unauthorized | Oh My Vue',
  },
};
</script>

<style scoped>
</style>
