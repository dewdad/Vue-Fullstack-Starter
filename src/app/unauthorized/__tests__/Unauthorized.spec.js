import { shallowMount } from '@vue/test-utils';

import Unauthorized from '../Unauthorized';

describe('Unauthorized', () => {
  let [wrapper] = [];

  beforeEach(() => {
    wrapper = shallowMount(Unauthorized);
  });

  it('should render an initial component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
