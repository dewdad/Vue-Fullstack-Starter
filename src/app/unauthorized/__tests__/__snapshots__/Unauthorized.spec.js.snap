// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized should render an initial component 1`] = `
<v-container fluid="" fill-height="">
  <v-layout justify-center="" align-center="">
    <div class="text-xs-center">
      <div class="display-2 primary--text pa-3">Whoops, 401!</div>
      <div class="pa-2">You don't have permission to go to this page.</div>
      <v-btn to="/" color="primary" outline="">Get me out of here!</v-btn>
    </div>
  </v-layout>
</v-container>
`;
